File: flight-ui/src/app/model/aircraft.ts
export class Aircraft {
  private code: String;
  private name: String;
  private model: String;

  public get $code(): String {
    return this.code;
  }

  public set $code(value: String) {
    this.code = value;
  }

  public get $name(): String {
    return this.name;
  }

  public set $name(value: String) {
    this.name = value;
  }

  public get $model(): String {
    return this.model;
  }

  public set $model(value: String) {
    this.model = value;
  }
}

File: flight-api/src/main/java/br/com/brsantiago/flightcontrol/model/domain/Aircraft.java
package br.com.brsantiago.flightcontrol.model.domain;


import javax.persistence.*;

/**
 * Created by bruno on 04/08/17.
 */
@Entity
public class Aircraft {
    @Id
    private int id;
    @Column(unique = true)
    private String code;
    private String name;
    private String model;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getModel() {
        return model;
    }

    public void setModel(String model) {
        this.model = model;
    }
}
